model/channel.go

```diff
@@ -188,6 +201,18 @@ func (o *Channel) IsValid() *AppError {
 	return nil
 }
 
+func (o *GetChannelsOptions) IsValid() *AppError {
+	if reflect.TypeOf(o.UserIds).String() != "[]string" {
```

Comment:
Can you elaborate why this is needed? The struct definition already ensures that `o.UserIds` is a string slice.